docs/instance_groups.md

```diff
@@ -549,3 +549,28 @@ spec:
   minSize: 2
   role: Node
 ```
+
+## Booting from a volume in OpenStack
+
+If you want to boot from a volume when you are running in openstack you can set annotations on the instance groups.
+
+```yaml
+# Example for nodes
+apiVersion: kops.k8s.io/v1alpha2
+kind: InstanceGroup
+metadata:
+  labels:
+    kops.k8s.io/cluster: k8s.dev.local
+  name: nodes
+  annotations:
+    openstack.kops.io/osVolumeBoot: enabled
+    openstack.kops.io/osVolumeSize: "15" # In gigabytes
```

Comment:
This should be specified as 15g 15t etc - defining it to always be gigabytes isn't future-proof

Reply:
Since OpenStack only accepts it is gigabytes and it is not a volume you would ever need to grow over 100G I went for a simpler implementation. I will look into addind support for annotations and default to gigabytes if none are provided.

Reply:
I have added support for it but now that I look at it it feels I just added an opportunity for bugs and maintenance overhead. Also it feels out of place as any other place you specify disk it is only in gigabytes.

Reply:
I would contact the actual kops maintainers and discuss with them which is better - being slightly more complex now to gain some form of longer-term viability or being simpler to maintain now at the cost of being limited to the sizes you can create

Reply:
I have messaged twice in the `kops-dev` channel about it but no one responded to it. Seeing as all other values for CPU/RAM/Disk marshal into ints I feel this is moving away from the standarts in kops. Also for this type of disk anything more then 100G is overkill and we are not limiting the amount they can request.They will be able to request a 1T if they want to.

Reply:
I reverted it to only Gigabytes. I think it is fine to submit it as it and if the kps teams have problems with it they can make a comment on it.